scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -17,11 +17,14 @@
 from __future__ import unicode_literals
 from __future__ import absolute_import
 
+import datetime
 import os
+import json
```

Comment:
Minor performance thing - we may want to use ``orjson`` library here (it's already bundled with the Windows binary).

Reply:
Ah thank you, will fix

Reply:
Hm it seems orjson is not supported with python 2.7: https://github.com/ijl/orjson/issues/35#issuecomment-545578561

Sticking with the standard library json module for now for the sake of the python 2.7 unittests.

Reply:
I guess, you also can use our wrapper json functions. They select json library itself and they have to use standard json lib as a fallback if `orjson` isn't fount.  https://github.com/scalyr/scalyr-agent-2/blob/ab6d933907884a9ccc61aa76e0857ec4d691a67b/scalyr_agent/util.py#L357-L386